lib/app.js

```diff
@@ -74,6 +74,11 @@ module.exports = settings => {
         { label: 'Downloads', href: '/downloads' }
       ];
     }
+
+    if (req.user.profile.asruRops) {
```

Comment:
Can we use `allowedActions` for this (as immediately above) so that if the logic changes in future it can be changed once?